build.gradle

```diff
@@ -162,12 +162,22 @@ task generateApiRequestFormat(type: cz.habarta.typescript.generator.gradle.Gener
     outputFile = "src/web/types/api-request.ts"
 }
 
+task generateIntent(type: cz.habarta.typescript.generator.gradle.GenerateTask) {
```

Comment:
You don't need to create a task. `generateApiRequestFormat` will help you. This indicates that you should move `Intent` to the correct folder.

Reply:
@xpdavid Understood. Removed `generateIntent` task

Reply:
@xpdavid Please review latest changes.